pkg/cli/kubectlwrappers/delete.go

```
@@ -0,0 +1,216 @@
+
+func runDeleteWithStorageConfirmation(cmd *cobra.Command, args []string, streams genericiooptions.IOStreams, originalRun func(*cobra.Command, []string), isTerminal bool) error {
+	opts, err := storageDeleteConfirmOptionsFromCmd(cmd, args, isTerminal)
+	if err != nil {
+		// Invalid flags such as --dry-run are reported by the wrapped command.
+		originalRun(cmd, args)
+		return nil
+	}
+
+	if opts.shouldWarn() {
+		if !confirmStorageDelete(streams.In, streams.ErrOut) {
+			fmt.Fprintf(streams.Out, "deletion is cancelled\n")
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintf` is not checked

(errcheck)